codegen/smithy-aws-typescript-codegen/src/main/resources/software/amazon/smithy/aws/typescript/codegen/sts-client-defaultStsRoleAssumers.ts

```diff
@@ -1,3 +1,5 @@
+// Please do not touch this file. It's generated from template:
+// https://github.com/aws/aws-sdk-js-v3/blob/main/codegen/smithy-aws-typescript-codegen/src/main/resources/software/amazon/smithy/aws/typescript/codegen/sts-client-defaultStsRoleAssumers.ts
```

Comment:
Add this message in codegen instead?
https://github.com/aws/aws-sdk-js-v3/blob/e6080b303b0934ecec36c9284e0530d4129bb79e/codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsAuthPlugin.java#L158-L167

The file inside codegen folder can be edited.

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/2360/commits/05d3c836de9a878752ab60ed765c9da6e56002e1